packages/fuels-core/src/types.rs

```diff
@@ -1,8 +1,10 @@
 use std::fmt;
 
-pub use fuel_tx::{Address, AssetId, ContractId, TxPointer, UtxoId};
```

Comment:
What is the substitute for `TxPointer` and `UtxoId` here?

Reply:
Good catch. It should be in `fuels::tx::`